packages/components/src/toggle-control/README.md

```diff
@@ -55,3 +55,10 @@ A function that receives the checked state (boolean) as input.
 - Type: `function`
 - Required: Yes
 
+### className
+
+The class that will be added with "components-base-control" and "components-toggle-control" to the classes of the wrapper div. If no className is passed only components-base-control and components-toggle-control are used.
```

Comment:
```suggestion
The class that will be added with `components-toggle-control` to the wrapper div. If no className is passed only `components-toggle-control` will be used.
```

Reply:
Doesn't the wrapper div have two classes, "components-base-control" and "components-toggle-control", as seen in my example above? 

Basically, I wrote the above description to be consistent with the documentation in packages/components/src/text-control/README.md where className adds a class in addition to "components-base-control" to the wrapper div:

> ### className
> 
> The class that will be added with "components-base-control" to the classes of the wrapper div. If no className is passed only components-base-control is used.

Reply:
You're right @AmartyaU! `components-base-control` is added by the `BaseControl` component and `components-toggle-control` by `ToggleControl`.

So the content of the README file looks good to me, but I'd suggest however to quote the class names with \` instead of with ". That way, they will be highlighted as code (see the [Markdown guide](https://guides.github.com/features/mastering-markdown/#syntax) for more context).

```suggestion
The class that will be added with `components-base-control` and `components-toggle-control` to the classes of the wrapper div. If no className is passed only `components-base-control` and `components-toggle-control` are used.
```

Reply:
@mmtr Thank you for confirming that the content of the README was accurate. I just updated the README with your suggested formatting changes!